spatialprofilingtoolbox/workflow/tabular_import/parsing/sample_stratification.py

```diff
@@ -1,5 +1,7 @@
 """Source file parsing regarding sample-level cohort identification."""
 import re
+from typing import Callable
+from typing import Any
```

Comment:
,

Reply:
If there are only two symbols imported, I'm not worried about creating an enumeration of them.

Reply:
A syntax that is dis-preferred is "from typing import Callable, Any". The only other syntax I would consider here is:
```py
from typing import (
    Callable,
    Any,
)
```
but that seems overkill for 2 items.